### DISCUSSION TITLE
W4 Combo option

### DISCUSSION THREAD

**Discussion by KirbyJoe** [TeamRizu/OutFox#350] [03 - Feature Requests]
In modern DDR games, the "Good" judgement also continues the combo. In StepMania, there are few themes to support W4 (Good/Decent) combos. For example, WAIEI2, a non-OutFox SM5 theme by waiei, has an option to change minimum judgement to continue the combo from W1 (Flawless/Marvelous/Fantastic) to W4 (Good/Decent). Using W4 combo option makes the gameplay more like current DDR games, and it's good for players who prefers modern DDR gameplay.

**Comment by Sudospective** [comment]
Isn't this something you can already do? If it can be controlled by themes, there must be some Lua API you can find that will allow you to change the combo falloff. I'd be surprised if I remember incorrectly about seeing it in the game settings, even.

**Reply by JoseVarelaP** [reply]
This is an option that themers can enable, by adding a setting to their metrics, it's not actually a game system option due to this.
```ini
[Gameplay]
MinScoreToContinueCombo=[The minimum TapNote Score Enumerator to be considered to continue]
```
